elastic/beats
# Metricbeat perfmon - fix issues with retrieving counter paths/adding counters to query when OS language is not ENG

Metricbeat perfmon - fix issues with retrieving counter paths/adding counters to query when OS language is is not ENG.

Should fix https://github.com/elastic/beats/issues/14684

Workaround follows also the NOTE steps in https://docs.microsoft.com/en-us/windows/win32/api/pdh/nf-pdh-pdhaddenglishcountera#remarks


LEFT TO DO
- [x]  Test performance
- [x] Test fix on Windows Server 2012

Reproduce steps:
- Set system language to other than ENG
- test the counter list will contain performance counter paths in the different language `TypePerf.exe –q > counters.txt`
- use both the English path and the counter path in the different lang when configuring windows.yml
- both options should work
